external-crates/move/move-compiler/src/editions/mod.rs

```diff
@@ -93,6 +94,7 @@ static SUPPORTED_FEATURES: Lazy<BTreeMap<Edition, BTreeSet<FeatureGate>>> =
     Lazy::new(|| BTreeMap::from_iter(Edition::ALL.iter().map(|e| (*e, e.features()))));
 
 const E2024_ALPHA_FEATURES: &[FeatureGate] = &[
+    FeatureGate::ConstInConst,
```

Comment:
Personally, I'm fine not feature gating this. Though I'm open to opinions.

Reply:
Feature gating removed! 👍